tailwind.config.cjs

```diff
@@ -26,6 +26,14 @@ module.exports = {
       transitionProperty: {
         width: 'width',
       },
+      colors: {
+        stopped: '#b6b6b6',
```

Comment:
I don't think we should add these generic names as it may not be the right name for example for another element

And in .js file we can use the same constant at different place.

I would add a new key and move the colors under this key.

https://tailwindcss.com/docs/customizing-colors

Key could be like 'containers' or other pattern but here I think bringing these colors at the top level is leading to case where we won't be able to reuse that name

so it would be like `bg-status-stopped`